Assets/Scripts/interact.cs

```diff
@@ -0,0 +1,68 @@
+using UnityEngine;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine.InputSystem;
+
+public class interact : MonoBehaviour
+{
+    public GameManager manager;
+    private GameObject obj;
+    public InputAction Use;
+    public GameObject text;
+    public GameObject player;
+    public bool inRange;
+    private void Start()
+    {
+        text.SetActive(false);
+    }
+    void Update()
+    {
+        if (inRange)
+        {
+            if (obj.name == "SceneSwitchDoor")
+            {
```

Comment:
CAPITALIZE THE FIRST LETTER FOR YOUR FILE.

Reply:
agreed